ui/manual.py

```diff
@@ -1,98 +1,100 @@
 from .braille import from_ascii
 from .book.book_file import BookFile, LoadState
+from .i18n import I18n
+import os
```

Comment:
This is an unused import. There are a [few other linter issues](https://travis-ci.org/Bristol-Braille/canute-ui/jobs/407299437#L528) which you can see by running `./lint`. You can auto-fix some of them by running `./fix` and you'll have to manually fix the rest. I would recommend running `./fix && ./lint` before `git commit`.